src/Core/src/ImageSources/FontImageSourceService/FontImageSourceService.Android.cs

```diff
@@ -7,13 +7,58 @@
 using Android.Graphics.Drawables;
 using Android.Util;
 using Bumptech.Glide;
+using Bumptech.Glide.Load;
+using Bumptech.Glide.Load.Engine;
+using Bumptech.Glide.Load.Resource.Bitmap;
+using Bumptech.Glide.Request;
+using Bumptech.Glide.Request.Target;
 using Microsoft.Extensions.Logging;
 using Microsoft.Maui.BumptechGlide;
 
 namespace Microsoft.Maui
 {
 	public partial class FontImageSourceService
 	{
+		public override async Task<IImageSourceServiceResult<Drawable>?> LoadDrawableAsync(IImageSource imageSource, Android.Widget.ImageView imageView, CancellationToken cancellationToken = default)
+		{
+			if (imageSource is IFontImageSource fontImageSource)
+			{
+				if (fontImageSource.IsEmpty)
+					return null;
+
+				var glyph = fontImageSource.Glyph;
+
+				var size = FontManager.GetFontSize(fontImageSource.Font);
+				var unit = fontImageSource.FontAutoScalingEnabled ? ComplexUnitType.Sp : ComplexUnitType.Dip;
```

Comment:
This uses `SP` units if we have font auto scaling enabled, otherwise just `DP`... The SP ones will scale with font accessibility settings on the device.